test/export/test_export.py

```diff
@@ -4736,6 +4736,138 @@ def forward(self, x, y):
         self.assertTrue(torch.allclose(orig_res[1], ep_res[1]))
         self.assertTrue(torch.allclose(orig_res[2], ep_res[2]))
 
+    def test_unflatten_placeholder_update_child2parent_swap(self):
```

Comment:
Can you add a test case dropout where it is forced to alias (in inference case this will happen)? This will make sure our mutation detecting thing is working fine.

Reply:
done